cranelift/filetests/filetests/runtests/popcnt.clif

```diff
@@ -4,6 +4,7 @@ target aarch64
 target s390x
 target x86_64
 target x86_64 has_popcnt
+;; target riscv64 i8x16 not supported by riscv64.
```

Comment:
Usually SIMD types are in separate test files. Can you move the `i8x16` test into a separate `simd-popcnt.clif` file and enable the scalar tests here?

Reply:
Yes,I can. But I have not edit this file before.